palace/linalg/hcurl.cpp

```diff
@@ -1,10 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-#include "curlcurl.hpp"
+#include "hcurl.hpp"
```

Comment:
This is referred to as an energy norm as well (most familiarly from solid mechanics at least where it more physically is kinetic + potential energy). So I renamed to that for clarity since indeed the coefficients are not one.

Reply:
Fixed back to `WeightedHCurlNorm` as per offline discussion, see [8f6c834](https://github.com/awslabs/palace/pull/64/commits/8f6c834e9d85533f75e93cb876b48cb8e4a9caff).